cifar/README.md

```diff
@@ -0,0 +1,177 @@
+# ExecuTorch Model Fine-Tuning on Android App
+
+In this tutorial, we will be fine-tuning a CIFAR 10 model on an android app using ExecuTorch.
+
+## Environment Setup
+
+For the android environment setup, follow these steps:
+
+1. Install Open JDK using: `brew install openjdk@17`
+2. Download and install the latest version of [Android Studio](https://developer.android.com/studio/)  .
+3. Start Android Studio and open the **Settings** dialog (gear icon on the bottom left).
+4. Navigate to **Languages & Frameworks**, then **Android SDK**.
+5. In the **SDK Platforms** tab, check **Android 14.0 (“UpsideDownCake”)** and **Android API 36**.
+6. In the **SDK Tools** tab, check **Android SDK Build-Tools 36**, **NDK (Side by side)**, **Android SDK Command-line Tools (latest)**, **CMake**, **Android Emulator**, and **Android SDK Platform-Tools**.
+7. Export the paths for these to your environment or add them to the `.bashrc` or `.zshrc` files:
+```bash
+# JAVA config
+export JAVA_HOME=/opt/homebrew/opt/openjdk@17/
+export PATH="$JAVA_HOME/bin:$PATH"
+
+# Android dev config
+export ANDROID_HOME=$HOME/Library/Android/sdk
+export PATH=$ANDROID_HOME/cmdline-tools/latest/bin:$PATH
+export PATH=$ANDROID_HOME/platform-tools:$PATH
+export PATH=$ANDROID_HOME/emulator:$PATH
+export PATH=$ANDROID_HOME/build-tools/latest:$PATH
+export ANDROID_NDK=$ANDROID_HOME/ndk/28.0.12433566/
+
+# Cmake config
+export PATH=$ANDROID_HOME/cmake/4.0.2/bin:$PATH
+```
+
+**NOTE** For the updated steps for building the dependencies refer to the official repository over [here](https://github.com/pytorch/executorch/blob/main/extension/android/README.md).
+
+To install ExecuTorch in a python environment we can use the following commands in a new terminal:
+
+```bash
+$ git clone https://github.com/pytorch/executorch.git
+$ cd executorch
+$ uv venv --seed --prompt et --python 3.10
```

Comment:
Not everyone has [uv](https://github.com/astral-sh/uv). We should suggest they download and use UV, or some other py env tool

Reply:
Oh I didn't know `uv` works on ExecuTorch